### DISCUSSION TITLE
Question Regarding Ensemble Mode

### DISCUSSION THREAD

**Discussion by ShaunKMan** [Anjok07/ultimatevocalremovergui#1511] [Q&A]
Hello,

I recently discovered UVR, and used a Youtube video as a guide to create some vocal stems (this free program does a better job than the fairly expensive Izotope RX 10 that I purchased, BTW).  

I downloaded a couple of models, as the video suggested, which gave him really good results.  Once the processing was complete, I ended up with four individual stems, which threw me off.

I guess based on the video I watched, I was assuming Ensemble mode 'combined' the selected models, as in the selected file would be processed through each method / model sequentially, resulting in a single stem.  Is this not the case?

Or in other words, using the example above, would I have achieved the same result had I chosen each specific process method / model, and just ran it four times (one for each method / model)?

I guess another possibility is I simply have no clue what I'm doing or talking about :-)

Anyways, any feedback is appreciated!

Shaun

**Comment by MrRorls12** [comment]
Use the "Vocals/Insturmentals" setting

**Comment by Gravitationalwavesarecool** [comment]
I dont use ensemble mode honestly, ive found its easier to learn what everything does trying them individually, and my results usually come out much cleaner using them one by one its more time consuming but worth it in my opinion

**Comment by ShaunKMan** [comment]
Thank you both for the replies.

I thought I *had* chosen the vocals/instrumentals option...but maybe not.

I'll be spending more time with the software to learn what works for me as well.

Still unclear on whether or not ensemble mode runs the file sequentially through each of the selected methods / models...can anyone shine some light in that regard?  Thanks again,

Shaun

**Reply by blissiraDC** [reply]
I'm wondering this also, did you ever find out?

**Reply by ShaunKMan** [reply]
No, not specifically, unfortunately.
